scripts/cococo/evaluations_movable_logical_qubits/fix_q_vary_depth/vary_depths_hex_251103.py

```
@@ -0,0 +1,317 @@
+sigma = 1
+n_circ = 10
+
+gates_list = [320, 640, 1280, 2560]
+depths_list = [40, 80, 160, 320]
+for num_gates, depth in zip(gates_list, depths_list, strict=False):
+    assert num_gates == depth * j, "depths list and gates_list does not coincide"
+gates_str = "_".join(map(str, gates_list))
+
+use_dag = True
+
+date_str = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)